aptos-move/move-examples/wrapped_token/sources/wrapped_token.move

```diff
@@ -0,0 +1,110 @@
+/// This provides an example for how to wrap a Token V1 in a Token V2
+module wrapped_token::wrapped_token {
+
+    use std::string::String;
+    use std::signer;
+    use aptos_token_objects::token::{create_from_account, generate_burn_ref, generate_mutator_ref};
+    use std::option;
+    use aptos_framework::object;
+    use std::option::Option;
+    use aptos_framework::object::{generate_extend_ref, generate_transfer_ref, ExtendRef, transfer, object_from_constructor_ref};
+    use aptos_token_objects::royalty;
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    /// Storage state for managing the no-code Token.
+    struct WrappedToken has key {
+        /// Token to wrap
+        token: 0x3::token::Token,
+        /// Used to burn.
+        burn_ref: Option<0x4::token::BurnRef>,
+        /// Used to control freeze.
+        transfer_ref: Option<object::TransferRef>,
+        /// Used to mutate fields
+        mutator_ref: Option<0x4::token::MutatorRef>,
+        /// Used to mutate fields
+        extend_ref: Option<ExtendRef>,
+    }
+
+    /// Create a collection based on the previous collection
+    ///
+    /// TODO: what should we do with the original collection?
+    public entry fun create_wrapped_collection(
+        new_creator: &signer,
+        creator_address: address,
```

Comment:
Might as well skip this param if it's expected to be the same as the signer. This is not subject to compat rule (being an example) so if we want new_creator != creator_address, we can add creator_address param in a later version